components/raftstore-v2/src/operation/ready/apply_trace.rs

```diff
@@ -809,6 +923,56 @@ mod tests {
         // Because modify is recorded, so we know there should be no admin
         // modification and index can be advanced.
         assert_eq!(5, trace.admin.flushed);
+
+        fn range_equals(pending_ranges: &VecDeque<IndexRange>, expected: Vec<IndexRange>) {
+            assert_eq!(pending_ranges.len(), expected.len());
+            pending_ranges
+                .iter()
+                .zip(expected.iter())
+                .for_each(|(r, e)| {
+                    assert_eq!(r, e);
+                });
+        }
+
+        trace.on_modify(CF_DEFAULT, 8);
+        let ingested_ssts_idx =
+            make_sst_apply_index(vec![(CF_DEFAULT, 6), (CF_WRITE, 6), (CF_DEFAULT, 7)]);
+        trace.on_sst_ingested(&ingested_ssts_idx);
+        trace.maybe_advance_admin_flushed(8);
+        assert_eq!(5, trace.admin.flushed);
```

Comment:
Follows above comment. Here take the DEFAULT as example. The [flushed, last_modified] is [5, 8] and we have default pending sst range [6, 8), so we can actually view DEFAULT has flushed_index 7 to be engaged in admin advance progress.

Reply:
Elaborate this case for each CF.
For CF_LOCK, it's [5, 5]. So does not block admin_index advance.
For CF_DEFAULT, it's [5, 8], but we have range[6, 8) which just follows flushed_index 5, so we can use 7 here.
For CF_WRITE, it's [5, 6] (trace.on_modify(CF_WRITE, 6) should have been called), but we  have range [6, 7), so we can deem it as [6, 6]. So does not block admin_index advance.
Therefore, we can advance admin_index to 7.

Reply:
done